test/upgrade.test.js

```diff
@@ -174,8 +174,7 @@ test('Dueling upgrade and response handling 1', function(t) {
         }
 
         try {
-            var upg = res.claimUpgrade();
-            upg.socket.destroy();
```

Comment:
This was removed because the `claimUpgrade()` call is what throws, so this line is never executed.